README.md

```diff
@@ -67,6 +69,8 @@ This tap:
    can be found. For example, it might look like:
    `https://mycompany.atlassian.net`.
 
+   The `groups` specifies groups for users stream. `groups` is optional parameter. Default value of groups is ["jira-administrators", "jira-software-users", "jira-core-users", "jira-users", "site-admin", "users"].
+
```

Comment:
I think the default value is as per below. https://github.com/singer-io/tap-jira/blob/41be5973fa4e6d33dc7e12bbff739c53c66a8581/tap_jira/streams.py#L230-L234

Reply:
Yea, I have updated the groups default values as shown in streams.py